automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala

```diff
@@ -221,7 +221,7 @@ class SnapshotAPISpec extends AnyFreeSpecLike with Matchers with BeforeAndAfterA
 
           // validate the snapshot was added correctly: list snapshots in Rawls, should return 1, which we just added.
           val listResponse = listSnapshotReferences(projectName, workspaceName)
-          val resources = Rawls.parseResponseAs[ResourceList](listResponse).getResources.asScala
+          val resources = Rawls.parseResponseAs[ResourceList](SnapshotListResponse).gcpDataRepoSnapshots
```

Comment:
shouldn't this be:
```
Rawls.parseResponseAs[SnapshotListResponse](listResponse).gcpDataRepoSnapshots
```

Reply:
Yes, this was a copy and paste mistake from my branch-switching shenanigans. I updated and re-ran the tests locally